src/portal/src/i18n/lang/zh-tw-lang.json

```diff
@@ -1420,427 +1419,427 @@
+    "FREE_ALL_SUCCESS": "成功釋放所有 worker",
+    "WORKER_POOL": "worker pool",
+    "WORKER_POOL_ID": "worker pool ID",
     "PID": "Pid",
-    "START_AT": "Started At",
+    "START_AT": "開始於",
     "HEARTBEAT_AT": "Heartbeat At",
-    "CONCURRENCY": "Concurrency",
-    "NO_WORKER_POOL": "We could not find any worker pool",
-    "FREE": "Free",
-    "WORKER_ID": "Worker ID",
-    "JOB_ID": "Job ID",
+    "CONCURRENCY": "並發性",
+    "NO_WORKER_POOL": "找不到任何worker pool",
+    "FREE": "釋放",
+    "WORKER_ID": "worker ID",
+    "JOB_ID": "工作 ID",
     "CHECK_IN_AT": "Checked In At",
-    "NO_WORKER": "We could not find any worker",
-    "JOB_QUEUE": "Job Queues",
-    "JOB_SERVICE_DASHBOARD": "Job Service Dashboard",
-    "OPERATION_STOP_ALL_QUEUES": "Stop all job queues",
-    "OPERATION_STOP_SPECIFIED_QUEUES": "Stop specified job queues",
-    "OPERATION_PAUSE_SPECIFIED_QUEUES": "Pause specified job queues",
-    "OPERATION_RESUME_SPECIFIED_QUEUES": "Resume specified job queues",
-    "OPERATION_PAUSE_SCHEDULE": "Pause all schedules",
-    "OPERATION_RESUME_SCHEDULE": "Resume all schedules",
-    "OPERATION_FREE_ALL": "Free all workers",
-    "OPERATION_FREE_SPECIFIED_WORKERS": "Free specified workers",
-    "QUEUE_STOP_BTN_INFO": "STOP — Stop and remove all jobs in the selected queue(s).",
-    "QUEUE_PAUSE_BTN_INFO": "PAUSE — Pause the execution of jobs in this type of job queue. Jobs can be enqueued when the queue is paused.",
-    "QUEUE_RESUME_BTN_INFO": "RESUME — Resume the execution of jobs in this type of job queue.",
-    "SCHEDULE_PAUSE_BTN_INFO": "PAUSE — Pause all schedules to execute.",
-    "SCHEDULE_RESUME_BTN_INFO": "RESUME — Resume all schedule to execute.",
-    "WORKER_FREE_BTN_INFO": "Stop the current running job to free the worker",
+    "NO_WORKER": "找不到任何worker",
+    "JOB_QUEUE": "工作佇列",
+    "JOB_SERVICE_DASHBOARD": "工作服務儀表板",
+    "OPERATION_STOP_ALL_QUEUES": "停止所有工作佇列",
+    "OPERATION_STOP_SPECIFIED_QUEUES": "停止指定的工作佇列",
+    "OPERATION_PAUSE_SPECIFIED_QUEUES": "暫停指定的工作佇列",
+    "OPERATION_RESUME_SPECIFIED_QUEUES": "恢復指定的工作佇列",
+    "OPERATION_PAUSE_SCHEDULE": "暫停所有排程",
+    "OPERATION_RESUME_SCHEDULE": "恢復所有排程",
+    "OPERATION_FREE_ALL": "釋放所有 worker",
+    "OPERATION_FREE_SPECIFIED_WORKERS": "釋放指定的 worker",
+    "QUEUE_STOP_BTN_INFO": "停止 - 停止並移除選定佇列中的所有工作。",
+    "QUEUE_PAUSE_BTN_INFO": "暫停 - 暫停此類型工作佇列中的工作執行。暫停時仍可將工作加入佇列。",
+    "QUEUE_RESUME_BTN_INFO": "繼續 - 恢復此類型工作佇列中的工作執行。",
+    "SCHEDULE_PAUSE_BTN_INFO": "暫停 - 暫停所有排程的執行。",
+    "SCHEDULE_RESUME_BTN_INFO": "繼續 - 恢復所有排程的執行。",
+    "WORKER_FREE_BTN_INFO": "停止目前執行的工作以釋放 worker",
     "CRON": "Cron",
-    "WAITING_TOO_LONG_1": "Certain jobs have been pending for execution for over 24 hours. Please check the job service ",
-    "WAITING_TOO_LONG_2": "dashboard.",
-    "WAITING_TOO_LONG_3": "For more details, please refer to the ",
-    "WAITING_TOO_LONG_4": "Wiki."
+    "WAITING_TOO_LONG_1": "某些工作等待執行超過 24 小時。請檢查工作服務儀表板。",
+    "WAITING_TOO_LONG_2": "更多詳細資訊，請參考 Wiki。",
```

Comment:
"dashboard"  and "Wiki" are hyperlink text， so they should be stand-alone items

Reply:
![Uploading image.png…]()

Reply:
Updated, thanks.